packages/@okta/vuepress-theme-prose/components/TermsAndConditionsDialog.vue

```diff
@@ -0,0 +1,215 @@
+    title="Tell us more about yourself"
+  >
+    <div class="dialog-text">
+      <p>
+        The information you provide will only be used to help us improve your
+        experience. Your personal data is safe and secure with us.
+      </p>
+      <label for="tac-country">
+        Country
+        <select
+          name="Country"
+          id="tac-country"
+          class="tac-row"
+          ref="countryEl"
+          v-model="selectedCountry"
+          @change="
+            region = $event.target.value;
+            resetFields();
+          "
+        >
+          <option disabled selected></option>
+          <option
+            v-for="country in countriesList"
+            v-bind:key="country.value"
+            :value="country.value"
+            >{{ country.name }}</option
+          >
+        </select>
+      </label>
+      <label for="tac-regionData" v-show="region.list.length">
+        {{ region.type }}
+        <select
+          name="Region"
+          id="tac-regionData"
+          class="tac-row"
+          ref="regionDataEl"
+          v-model="selectedRegion"
+        >
+          <option disabled selected></option>
+          <option
+            v-for="regionData in region.list"
+            v-bind:key="regionData.name"
+            :value="regionData.value"
+            >{{ regionData.name }}</option
+          >
+        </select>
+      </label>
+      <div class="marketing-c" v-show="isGDPRCountry">
+        <div class="title">
+          Marketing Communication <span> - Optional</span>
+        </div>
+        <div class="gdpr-condition">
+          <input
+            type="checkbox"
+            name=""
+            id="okta-contact"
+            ref="gdprBoxEl"
+            checked="false"
+          />
+          <label for="okta-contact">
+            I agree that Okta may contact me with marketing communications. See
+            Privacy Policy for details on how to unsubscribe.
+          </label>
+        </div>
+      </div>
+    </div>
+    <template v-slot:footer>
+      <div class="agree-policy" v-show="selectedCountry">
+        <div class="splitter"></div>
+        <p>
+          By clicking "continue", I agree to the applicable Free Trial terms in
+          <SmartLink :item="{ link: '/terms/' }">Okta's Terms</SmartLink> and
+          the processing of my personal data by Okta as described in the
+          <SmartLink :item="{ link: '/privacy-policy/' }"
+            >Privacy Policy</SmartLink
+          >.
+        </p>
+      </div>
+      <div class="terms-conditions-btns">
+        <div class="btn-container">
+          <input
+            type="button"
+            class="btn social-btn"
+            value="cancel"
+            @click="closeTermsConditionsDialog()"
+          />
+        </div>
+        <div class="btn-container">
+          <input
+            type="button"
+            class="btn red-button"
+            @click="!isDisabledSocialAuth && setTaCUrlAndRedirect()"
```

Comment:
Is the `!isDisabledSocialAuth` part necessary if the input is disabled via the `:disabled` property below?

Reply:
it's a protecting mechanism if someone will want to remove disabled attr maually from dev console. Not sure about prod build but in dev env removing disabled attr will allow executing onclick  event